melusine/models/train.py

```diff
@@ -214,7 +214,8 @@ def __getstate__(self):
         dict_attr = dict(self.__dict__)
         if "model" in dict_attr:
             del dict_attr["model"]
-            del dict_attr["embedding_matrix"]
+            if "embedding_matrix" in dict_attr:
```

Comment:
I think the condition:
`if "embedding_matrix" in dict_attr`
Could have the same indentation as :
`if "model" in dict_attr`

(The second if statement is not dependent on the first one)